### DISCUSSION TITLE
Help needed , no one is responding!

### DISCUSSION THREAD

**Discussion by hkiiita** [community/community#25383] [Actions]
<p>HI ,</p>
<p>Apologies for above heading but no one is responding in Github Accounts thread hence posting here.</p>
<p>I had recieved an authorised email that i have been granted access to student developer pack {after i had sent my college ID etc as asked}, but am unable to access it , i mean when i visit <a href="https://education.github.com/pack" rel="nofollow noopener">https://education.github.com/pack </a> i dont see a login button anywhere though i am logged in to my github account in another tab in same browser.</p>
<p>If i look my <a href="https://github.com/settings/billing" rel="nofollow noopener">https://github.com/settings/billing</a> page i see</p>
<ol>
<li>GitHub Pro<br>
2)GitHub Learning Lab<br>
3)GitHub Actions<br>
4)GitHub Packages</li>
</ol>
<p>but NOT Github Studertn developer pack …</p>
<p>what am i doing wrong ?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi <a class="mention" href="/hkbiet">@hkbiet</a>,<br>
GitHub Actions category is not a right place for your question. Please try to contact Education Support here : <a href="https://education.github.com/contact" rel="nofollow noopener">https://education.github.com/contact</a></p>
